clang/test/Preprocessor/init.c

```diff
@@ -2742,3 +2742,275 @@
+// XTENSA: #define __UINT64_TYPE__ long long unsigned int
+// XTENSA: #define __UINT8_C_SUFFIX__ 
+// XTENSA: #define __UINT8_MAX__ 255
+// XTENSA: #define __UINT8_TYPE__ unsigned char
+// XTENSA: #define __UINTMAX_C_SUFFIX__ ULL
+// XTENSA: #define __UINTMAX_MAX__ 18446744073709551615ULL
+// XTENSA: #define __UINTMAX_TYPE__ long long unsigned int
+// XTENSA: #define __UINTMAX_WIDTH__ 64
+// XTENSA: #define __UINTPTR_MAX__ 4294967295U
+// XTENSA: #define __UINTPTR_TYPE__ unsigned int
+// XTENSA: #define __UINTPTR_WIDTH__ 32
+// XTENSA: #define __UINT_FAST16_MAX__ 65535
+// XTENSA: #define __UINT_FAST16_TYPE__ unsigned short
+// XTENSA: #define __UINT_FAST32_MAX__ 4294967295U
+// XTENSA: #define __UINT_FAST32_TYPE__ unsigned int
+// XTENSA: #define __UINT_FAST64_MAX__ 18446744073709551615ULL
+// XTENSA: #define __UINT_FAST64_TYPE__ long long unsigned int
+// XTENSA: #define __UINT_FAST8_MAX__ 255
+// XTENSA: #define __UINT_FAST8_TYPE__ unsigned char
+// XTENSA: #define __UINT_LEAST16_MAX__ 65535
+// XTENSA: #define __UINT_LEAST16_TYPE__ unsigned short
+// XTENSA: #define __UINT_LEAST32_MAX__ 4294967295U
+// XTENSA: #define __UINT_LEAST32_TYPE__ unsigned int
+// XTENSA: #define __UINT_LEAST64_MAX__ 18446744073709551615ULL
+// XTENSA: #define __UINT_LEAST64_TYPE__ long long unsigned int
+// XTENSA: #define __UINT_LEAST8_MAX__ 255
+// XTENSA: #define __UINT_LEAST8_TYPE__ unsigned char
+// XTENSA: #define __USER_LABEL_PREFIX__ 
+// XTENSA: #define __WCHAR_MAX__ 2147483647
+// XTENSA: #define __WCHAR_TYPE__ int
+// XTENSA: #define __WCHAR_WIDTH__ 32
+// XTENSA: #define __WINT_MAX__ 4294967295U
+// XTENSA: #define __WINT_TYPE__ unsigned int
+// XTENSA: #define __WINT_UNSIGNED__ 1
+// XTENSA: #define __WINT_WIDTH__ 32
+// XTENSA: #define __XCHAL_HAVE_ABS 1
+// XTENSA: #define __XCHAL_HAVE_ADDX 1
+// XTENSA: #define __XCHAL_HAVE_BE 0
+// XTENSA: #define __XCHAL_HAVE_BOOLEANS 0
+// XTENSA: #define __XCHAL_HAVE_DEBUG 0
+// XTENSA: #define __XCHAL_HAVE_DENSITY 0
+// XTENSA: #define __XCHAL_HAVE_DIV32 0
+// XTENSA: #define __XCHAL_HAVE_FP 0
+// XTENSA: #define __XCHAL_HAVE_FP_DIV 0
+// XTENSA: #define __XCHAL_HAVE_FP_POSTINC 0
+// XTENSA: #define __XCHAL_HAVE_FP_RECIP 0
+// XTENSA: #define __XCHAL_HAVE_FP_RSQRT 0
+// XTENSA: #define __XCHAL_HAVE_FP_SQRT 0
+// XTENSA: #define __XCHAL_HAVE_L32R 1
+// XTENSA: #define __XCHAL_HAVE_LOOPS 0
+// XTENSA: #define __XCHAL_HAVE_MAC16 0
+// XTENSA: #define __XCHAL_HAVE_MINMAX 0
+// XTENSA: #define __XCHAL_HAVE_MUL16 0
+// XTENSA: #define __XCHAL_HAVE_MUL32 0
+// XTENSA: #define __XCHAL_HAVE_MUL32_HIGH 0
+// XTENSA: #define __XCHAL_HAVE_NSA 0
+// XTENSA: #define __XCHAL_HAVE_S32C1I 0
+// XTENSA: #define __XCHAL_HAVE_SEXT 0
+// XTENSA: #define __XCHAL_HAVE_THREADPTR 0
+// XTENSA: #define __XCHAL_HAVE_WINDOWED 0
+// XTENSA: #define __XTENSA_CALL0_ABI__ 1
+// XTENSA: #define __XTENSA_EL__ 1
+// XTENSA: #define __XTENSA_SOFT_FLOAT__ 1
+// XTENSA: #define __XTENSA__ 1
+// XTENSA: #define __xtensa__ 1
```

Comment:
Please add the newline back to the end of the file.

Reply:
fixed